backend/src/v5/middleware/dataConverter/outputs/teamspaces/projects/models/commons/tickets.js

```diff
@@ -95,9 +98,37 @@ Tickets.serialiseTicket = async (req, res) => {
 	try {
 		const { teamspace } = req.params;
 		const { ticket, showDeprecated } = req;
-		const template = generateFullSchema(await getTemplateById(teamspace, ticket.type));
 
-		respond(req, res, templates.ok, serialiseTicket(ticket, template, !showDeprecated));
+		respond(req, res, templates.ok, await serialiseTicket(teamspace, ticket, undefined, !showDeprecated));
+	} catch (err) {
+		respond(req, res, templates.unknown);
+	}
+};
+
+const getTemplatesDictionary = async (teamspace, templateIds) => {
+	const res = {};
+
+	const data = await getTemplatesByQuery(teamspace, { _id: { $in: templateIds } });
```

Comment:
would probably prefer not to because:

1. that's pretty much all the documents anyway
2. the logic is that we are fetching the template to give it to another function to give us the full template, it's a bit high coupling for this middleware to know what `generateFullSchema` would require.
3. it'd be prone to error, if we ever extend the schema we won't remember that we did a projection here and would start wondering where the data went.


So I think this change is only sound if generateFullSchema does the database call to get the data, as that is the function that is working on the data, it knows what it needs. `getTemplatesDictionary` doesn't. But we end up with unnecessary database handler dependency on the schema file.

all in all, not worth it to save a couple of bytes.